platform/commonUI/general/src/directives/MCTSplitter.js

```diff
@@ -56,6 +56,12 @@ define(
 
                         // Update the position of this splitter
                         mctSplitPane.position(initialPosition + pixelDelta);
+
+                        // Save user width preference to local storage if selected
+                        if (mctSplitPane.saveUserWidthPreference === 'true') {
```

Comment:
Can you have MCTSplitter notify MCTSplitPane on change and have MCTSplitPane handle updating localStorage?  i.e. e.g. `mctSplitPane.onChange(initialPosition + pixelDelta);`  

That keeps the responsibilities in one place and will simplify your testing.